config/peerpods/podvm/packer-resource-cleanup.sh

```diff
@@ -98,12 +97,48 @@ function azure_cleanup() {
     done
 }
 
+function aws_cleanup() {
+    log "Deleting Packer VMs"
+
+    aws ec2 describe-instances --filters "Name=instance.group-name,Values=packer*" \
```

Comment:
Is this security group (packer*) used exclusively by us, or could it be shared with other Packer builds on the same cluster? Just making sure we’re not risking unintended deletions.

Reply:
Packer creates a dynamic SG with the prefix `packer`.

Reply:
Yes, but what if the user is using packer for something else? would we be removing that? Are we ok with that approach?

Reply:
I think it's fine for our use case. As part of the podvm creation a default vpc is created and packer vm is instantiated there.  Let me add a filter also for the default VPC and force push. 
I'm open for any other ideas as well

Reply:
Adding tag is the best approach. To do that, we need to add changes to CAA repo or maintain the changes locally in OSC repo.
Btw, the VPC is created by the packer generation script itself. So that's fine as well imho.
Let me know what you prefer. I will be able to get back to this most likely after Kubecon  EU.

Reply:
Well are you talking about the `cloud-api-adaptor/src/cloud-api-adaptor/aws/image/Makefile`? If so does the sandboxed-containers-operator allow setting a custom VPC, because in such case we'd have to query that, if specified, rather than the default VPC.

As for the `packer-` instances, it's possible the user uses another `packer-` ones so it'd be nicer to add a custom (ideally unique and random, or at least CAA specific) tag to minimize this problem, but the likeliness of this shouldn't be great. So for the initial version we might get away with just documenting it?

Reply:
OSC doesn't allow setting of a custom VPC. During build process it will create default-vpc and use it
https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/aws/image/Makefile#L39C2-L39C59
Ideally, we should tag the resources. But imho this is not really a blocker. We can document this aspect.

Reply:
I do agree and provided we can not supply a custom VPC it should serve well. Thanks